### DISCUSSION TITLE
Using my own weights and images in Docker

### DISCUSSION THREAD

**Discussion by Abzurde** [ultralytics/yolov5#8131] [Q&A]
Hi, I'm a new docker user, and I wanted to know, is there any way I can import my own weights that I trained and my images inside the container ? (then use detect.py, I don't need to train or val)

**Comment by glenn-jocher** [comment]
@Abzurde sure, you can use the volumes argument when creating a container to access local files. See https://docs.docker.com/storage/volumes/

**Reply by Abzurde** [reply]
Thank you !
